src/platform/src/main/java/org/geoserver/platform/ModuleCapabilities.java

```diff
@@ -0,0 +1,12 @@
+package org.geoserver.platform;
+
+public interface ModuleCapabilities {
```

Comment:
Actually why an interface? You always end up using Capability, why not creating an enum named ``ModuleCapbility`` and use everywhere?

If you want to vouch for the future, you could create an interface named ``ModuleCapability`` and the create the enum ``Capability`` inside it an make it implement the ``ModuleCapability`` interface. For no the ``ModuleCapability`` interface could have the ``String getName()`` method. Makes sense? (Maybe I missed the propose of the current interface).

Reply:
fixed